build/package_release.php

```diff
@@ -156,7 +156,7 @@
 system('zip -r ../packages/'.$appVersion.'.zip . -x@../exclude_files.txt -x@../exclude_files_full.txt > /dev/null');
 
 echo "Packaging Mautic Update Package\n";
-system('zip -r ../packages/'.$appVersion.'-update.zip -x@../exclude_files.txt -@ < modified_files.txt > /dev/null');
+system('zip -r ../packages/'.$appVersion.'-update.zip -x@../exclude_files.txt -@ < modified_files.txt -@ < deleted_files.txt > /dev/null');
```

Comment:
This leads to the following error:

```
Packaging Mautic Full Installation
Packaging Mautic Update Package
4.0.0-beta-update.zip: No such file or directory
SHA1(4.0.0-beta.zip)= 374ebc46d800e8037e89567bde90cbafcd64034a
dennis@mautic-web:/var/www/html$ 
```

Removing the `/dev/null` at the end gives some more details:

```
zip error: Invalid command arguments (nothing to select from)
```

To exclude the files in `deleted_files.txt`, I believe the correct syntax is `-x@deleted_files.txt`.

Let's take this opportunity to improve this line a bit more:
- Add `-q` at the beginning (causes the `zip` command to be quiet and not output every single file it's including in the ZIP)
- Remove `/dev/null` from the end. No longer needed due to `-q` and helps us in the future by showing actual errors instead of forwarding them to `/dev/null`
- Move `-x@deleted_files.txt` right after `-x@../exclude_files.txt` for slightly better readability

```suggestion
system('zip -qr ../packages/'.$appVersion.'-update.zip -x@../exclude_files.txt -x@deleted_files.txt -@ < modified_files.txt');
```